mne/dipole.py

```diff
@@ -41,15 +42,15 @@ class Dipole(object):
     Parameters
     ----------
     times : array, shape (n_dipoles,)
-        The time instants at which each dipole was fitted.
+        The time instants at which each dipole was fitted (sec).
     pos : array, shape (n_dipoles, 3)
         The dipoles positions in meters
     amplitude : array, shape (n_dipoles,)
-        The amplitude of the dipoles in nAm
+        The amplitude of the dipoles (nA).
```

Comment:
Ahh, I didn't realize it was a dipolar moment, I had thought of it as an actual current... will fix.